frontend/src/metabase/actions/containers/ActionCreatorRoute/ActionCreatorRoute.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import type { Location } from "history";
+
+import * as Urls from "metabase/lib/urls";
+import Models from "metabase/entities/questions";
+
+import type { Card, WritebackActionId } from "metabase-types/api";
+import type { State } from "metabase-types/store";
+
+import ActionCreator from "../ActionCreator";
+
+interface OwnProps {
+  location: Location;
+  params: {
+    slug: string;
+    actionId?: WritebackActionId;
```

Comment:
is `params` coming from `react-router`? If that's the case I believed `actionId` would be a string.

**edited** Just tested, it's confirmed they're all string
<img width="164" alt="image" src="https://user-images.githubusercontent.com/1937582/218538944-17afa137-558f-46a0-9985-299d38723e71.png">

Reply:
Oh right! Fixed, thank you